regression/ansi-c/gcc_attributes5/main.c

```diff
@@ -3,21 +3,29 @@
 
 #ifdef __GNUC__
 
-char __attribute__((aligned(8))) var1;
-__attribute__((aligned(8))) char var2;
-char var3 __attribute__((aligned(8)));
-int (__attribute__((aligned(8))) var4);
-int (__attribute__((aligned(8))) (var5));
-int (__attribute__((aligned(8))) *var6);
-int __attribute__((aligned(8))) *var7;
+char __attribute__((aligned(16))) var1;
```

Comment:
I don't really understand why various alignments are changed from 8 to 16? All I would have expected is the `#ifdef __clang` part below.

Reply:
The reason that the test has passed so far was a bit random: the default alignment of pointers on 64-bit machines is 8. Using 16 avoids that coincidence.

Reply:
Ok, could you please add that to the commit message?

Reply:
Done